docs/AndroidBuildingFromSource.md

```diff
@@ -20,13 +20,15 @@ Make sure you have the following installed:
 3. Local Maven repository for Support Libraries (formerly `Android Support Repository`) >= 17 (for Android Support Library)
 4. Android NDK (download links and installation instructions below)
 
-Point Gradle to your Android SDK: either have `$ANDROID_SDK` and `$ANDROID_NDK ` defined, or create a local.properties file in the root of your react-native checkout with the following contents:
+Point Gradle to your Android SDK: either have `$ANDROID_SDK` and `$ANDROID_NDK ` defined in your bash profile, or create a local.properties file in the `android` directory of your react-native app with the following contents:
```

Comment:
I've often found that both are necessary when you're using android studio.

BTW, if we're mentioning bash, I think it'll be useful to point to the file path itself, `~/.bash_profile`. We shouldn't ignore other shells like `zsh` either though. So a sentence for other shells might be useful too.